config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/loader/EncassLoader.java

```diff
@@ -54,9 +55,13 @@ public void load(Bundle bundle, Element element) {
 
     private String getPath(Bundle bundle, String policyId) {
         List<Policy> policyList = bundle.getPolicies().values().stream().filter(p -> policyId.equals(p.getId())).collect(Collectors.toList());
+        if ((policyList.isEmpty() || policyList.size() > 1) && bundle.getLoadingMode() == PERMISSIVE) {
```

Comment:
Is it fine to use == for enums?

Reply:
Yes enums are singleton objects in the JVM so there is always only one instance of each enum value and it's safe to compare by ==